### ISSUE TITLE
[OPS] Unable to change server logo

### ISSUE THREAD

**Issue by alexxxmendonca** [pkp/pkp-lib#5650]
**Describe the bug**
As a Server Administrator, I am unabled to change the Server Logo. It keeps loading _ad eternum_ and the logo doesn't change. The system freezes and I'm unabled to do anything else.

**To Reproduce**
![Website Settings](https://user-images.githubusercontent.com/4059679/77167892-a0e11b80-6a95-11ea-9a82-fd7f11d7b4ed.gif)

**What application are you using?**
OPS 3.2.0 Beta

**Comment by asmecher** [comment]
@alexxxmendonca, check the server's error log for details.

**Comment by alexxxmendonca** [comment]
Our tech guy was unable to assist. According to him, there are no errors.

I should add that, even though there is no UI feedback and the page continues to load _ad eternum_, the logos are changed if you refresh the public page.

**Comment by ajnyga** [comment]
Can not reproduce this with current master. Can you @asmecher?

**Comment by asmecher** [comment]
@alexxxmendonca, I wonder if your tech guy was looking in the wrong log? This [other issue](https://github.com/pkp/pkp-lib/issues/5651) should definitely have raised an error message in the log, but he didn't find it.

**Comment by ajnyga** [comment]
I do not think this is a logo upload issue. The upload there works, but saving the form creates an error. So could be that other data on that form does it. Could be that it is the same issue as https://github.com/pkp/pkp-lib/issues/5651

**Comment by alexxxmendonca** [comment]
The behavior is the same, yes. And the logo does change, you just have to force a stop.

**Comment by ajnyga** [comment]
Yes, it occurs when you have content in the Page footer field.

**Comment by ajnyga** [comment]
The same error:

```
[23-Mar-2020 16:06:40 Europe/Berlin] PHP Fatal error:  Uncaught Exception: DB Error: Unknown column 'setting_type' in 'field list' Query: INSERT INTO journal_settings
					(journal_id, setting_name, setting_value, setting_type, locale)
					VALUES (?, ?, ?, ?, ?) in /Applications/XAMPP/xamppfiles/htdocs/ops-test/lib/pkp/classes/db/DAO.inc.php:703
```

**Comment by ajnyga** [comment]
After finding out some exciting new things about browser caching, I can conclude that https://github.com/ajnyga/ojs/commit/0b3b9cdcf0bc4fe62169cbe0d41bd8dd6c1366bb is a fix for this as well.

**Comment by asmecher** [comment]
Thanks, @ajnyga! @alexxxmendonca, this will be released in the OPS 3.2.0-1 build, likely sometime this week.
